challenge.rb

```diff
@@ -1,26 +1,34 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map {|str| str.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  [input.fetch(2)]
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  arr = []
+  input.each do |str|
+    if str != 'dog'
+      arr.push str
+    end
+  end
+  arr
 end
```

Comment:
You're right! I was trying to only use the enumerable methods once throughout... But then redid that method later on - I'll change it here